terraform/README.md

```diff
@@ -109,6 +111,7 @@ future changes by simply running `terraform apply
 |------|-------------|
 | access_terraform_backend_role | The IAM role that allows sufficient access to the Terraform S3 bucket and DynamoDB table to use those resources as a Terraform backend. |
 | provisionaccount_role | The IAM role that allows sufficient permissions to provision all AWS resources in the Terraform account. |
+| read_terraform_state_role | The IAM role that allows read-only access to the S3 bucket where Terraform state is stored. |
```

Comment:
I would have called this `terraform_state_readonly_role`, but I won't quibble.

Reply:
I thought that `read_terraform_state_role` was more aligned with the `access_terraform_backend_role` that we already had.